src/metabase/db.clj

```diff
@@ -49,7 +49,10 @@
   "Parse a DB connection URI like `postgres://cam@localhost.com:5432/cams_cool_db?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory` and return a broken-out map."
   [uri]
   (when-let [[_ protocol user pass host port db query] (re-matches #"^([^:/@]+)://(?:([^:/@]+)(?::([^:@]+))?@)?([^:@]+)(?::(\d+))?/([^/?]+)(?:\?(.*))?$" uri)]
-    (merge {:type     (keyword protocol)
+    (merge {:type     (case (keyword protocol)
```

Comment:
I think MB_DB_CONNECTION_URI worked for other databases besides Postgres and MySQL, so this will break those. Wouldn't it be better to do something like

```
(case (keyword protocol)
      :postgresql :postgres
      (keyword protocol))
```

so we don't have to add cases for the other dbs?